src/utils/util.ts

```diff
@@ -86,15 +86,17 @@ export const isValidReactNode = (node: ReactNode) => {
   return !!node
 }
 
-export const matchScript = (contractHash: string, hashType: string): ContractHashTag | undefined => {
-  if (isMainnet()) {
-    return MainnetContractHashTags.find(
-      scriptTag => scriptTag.codeHashes.find(codeHash => codeHash === contractHash) && scriptTag.hashType === hashType,
-    )
-  }
-  return TestnetContractHashTags.find(
-    scriptTag => scriptTag.codeHashes.find(codeHash => codeHash === contractHash) && scriptTag.hashType === hashType,
+export const matchScript = (script: State.Script): ContractHashTag | undefined => {
+  const matchedScript = (isMainnet() ? MainnetContractHashTags : TestnetContractHashTags).find(
+    scriptTag =>
+      scriptTag.codeHashes.find(codeHash => codeHash === script.codeHash) && scriptTag.hashType === script.hashType,
   )
+  // Multi-signature locking with time limit: unlock after a specified on-chain time (timestamp, block)
+  // The args length is 20bytes without time lock, and 28bytes with time lock.
+  if (matchedScript && matchedScript.tag === 'secp256k1 / multisig' && script.args.length === 28 * 2 + 2) {
```

Comment:
You should define a variable named `SECP256K1_MULTISIG_TAG` in `scripts.ts`, with the value `"secp256k1 / multisig"`.
This helps to prevent potential issues during future maintenance.

Reply:
> You should define a variable named `SECP256K1_MULTISIG_TAG` in `scripts.ts`, with the value `"secp256k1 / multisig"`. This helps to prevent potential issues during future maintenance.

but scripts defined in `scripts.ts` will show at `https://explorer.nervos.org/scripts`, Otherwish I append some fields to `ContractHashTag`
- `showInScripts: boolean`  is this script show in `https://explorer.nervos.org/scripts` page
- `matchArgs: (args: string) => boolean` for tags show, match the args

how about this solution

Reply:
I didn't understand what you meant, and perhaps you didn't understand my point either.
Let me provide an example:

```
export const SECP256K1_MULTISIG_TAG = 'secp256k1 / multisig'

export const MainnetContractHashTags: ContractHashTag[] = [
  ...,
  {
    codeHashes: ['0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8'],
    txHashes: ['0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c-1'],
    depType: 'dep_group',
    hashType: 'type',
    tag: SECP256K1_MULTISIG_TAG,
    category: 'lock',
  },
  ...
]

export const TestnetContractHashTags: ContractHashTag[] = [
  ...,
  {
    codeHashes: ['0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8'],
    txHashes: ['0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37-1'],
    depType: 'dep_group',
    hashType: 'type',
    tag: SECP256K1_MULTISIG_TAG,
    category: 'lock',
  },
  ...,
]
```

Reply:
I understand your solution, I mean that 
```
  {
    codeHashes: ['0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8'],
    txHashes: ['0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c-1'],
    depType: 'dep_group',
    hashType: 'type',
    tag: 'secp256k1 / multisig',
    category: 'lock',
  },
  {
    codeHashes: ['0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8'],
    txHashes: ['0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c-1'],
    depType: 'dep_group',
    hashType: 'type',
    tag: 'secp256k1 / multisig / locktime',
    category: 'lock',
  },
  ```
  this will cause the script page display two different Script, but they have the same infos
  
<img width="1076" alt="image" src="https://github.com/nervosnetwork/ckb-explorer-frontend/assets/7877698/c96bd500-f88d-4b32-8ce4-86841b60d3bd">